packages/mc-scripts/src/commands/start-vite.ts

```diff
@@ -46,6 +46,9 @@ async function run() {
       pluginSvgr(),
       pluginCustomApplication(applicationConfig),
     ],
+    optimizeDeps: {
```

Comment:
I'm having problems checking this locally, but I'd like to share my opinion anyway: if we configure it like this, I believe it will be harder for us to keep track of the workaround. I mean, it is a change in a file that we don't change quite often. On the other hand, if we downgrade `vite` version, even `renovate` bot will warn us about it so we can have a PR where we can add a label "blocking" it with a comment describing why.

Reply:
Thanks. Np.
If this doesn't work then we can downgrade vite. I felt that downgrading would be a last resort.